apps/web/vibes/soul/primitives/product-card/index.tsx

```diff
@@ -84,9 +84,15 @@ export function ProductCard({
   );
 }
 
-export function ProductCardSkeleton({ className }: { className?: string }) {
+export function ProductCardSkeleton({
+  className,
+  loading = true,
+}: {
+  className?: string;
+  loading?: boolean;
+}) {
   return (
-    <div className={clsx('animate-pulse', className)}>
+    <div className={clsx(loading && 'animate-pulse', className)}>
```

Comment:
Not something we need to address in this PR but I pattern I saw in Catalyst `main` that I liked was using `data-pending` in various components and then using Tailwind `group` at the top level and then applying the class `group-has-[[data-pending]]:animate-pulse` at the top-level as well

Reply:
https://github.com/bigcommerce/catalyst/blob/d38f164d3e87ca87d3e792f8058a74c1f13e4220/core/app/%5Blocale%5D/(default)/(faceted)/search/page.tsx#L88

Reply:
I was having some issues getting `group-has` to work, but I included a different version that uses `group-data-[pending]`.